drivers/network/tcpip/tcpip/iinfo.c

```diff
@@ -83,9 +94,14 @@ TDI_STATUS InfoTdiQueryGetInterfaceMIB(TDIEntityID ID,
                     case NdisPhysicalMedium1394:
                     case NdisPhysicalMediumWirelessWan:
 #endif
+#if (_WIN32_WINNT >= _WIN32_WINNT_VISTA)
```

Comment:
Is this `#if` actually needed/wanted? (It looks like to me it might be superfluous, even counter-productive.)

Reply:
Is it better if we'll compile this for NT5 too, when it isn't actually needed in that case? Isn't it a wasting of compile time and binary size?

Reply:
This `#if` is correct as is.
I am simply wondering whether not adding it could maybe help support a "NT6+" driver, on our NT5.2 kernel ?

Reply:
>I am simply wondering whether not adding it could maybe help support a "NT6+" driver, on our NT5.2 kernel ?

Not sure about that at all. Most likely it will not help and even not change anything anyway, because NT6+ networking miniport drivers (if you mean them of course) need much more NT6 stuff like NDIS 6.0+, so we need to export much more Win32/NTAPI stuff and have NT6-compatible networking stack implementation, while we currently have NT 5.2 compatible NDIS only.